src/api/app/views/shared/_buildresult_box.html.haml

```diff
@@ -0,0 +1,88 @@
+    %li.selected
+      %a{class: "result_select_link_#{index}", href: '#', id: "result_select_link_#{index}_0"} Build Results
+      = sprite_tag('reload', title: 'Reload', onclick: 'update_build_result()', class: "result_reload_#{index}", id: "result_reload_#{index}_0")
+      = image_tag('ajax-loader.gif', id: "result_spinner_#{index}_0")
+    - if controller == 'package'
+      %li
+        %a{class: "result_select_link_#{index}", href: '#', id: "result_select_link_#{index}_1"} Rpmlint Results
+        = sprite_tag('reload', title: 'Reload', onclick: 'update_rpmlint_result()', class: "result_reload_#{index}", id: "result_reload_#{index}_1", style: 'display: none')
+        = image_tag('ajax-loader.gif', id: "result_spinner_#{index}_1", class: 'hidden')
+%div{class: "result_display_#{index}", id: "result_display_#{index}_0"}
+- if controller == 'package'
+  %div{class: "result_display_#{index}", id: "result_display_#{index}_1"}
+= javascript_tag do
+  - if controller == 'package'
+    :plain
+      $('.result_select_link_#{index}').click(function (event) {
+        $('#result_select_#{index} li.selected').attr('class', '');
+        $(event.target).parent().attr('class', 'selected')
+        $('.result_display_#{index}').hide();
+        $('.result_reload_#{index}').hide();
+        var tab_index = event.target.id.split('result_select_link_#{index}_')[1];
+        $('#result_reload_#{index}_' + tab_index).show();
+        $('#result_display_#{index}_' + tab_index).show();
+        return false;
+      });
+
+      function update_rpmlint_result() {
+        $('#result_spinner_#{index}_1').show();
+        $('#result_reload_#{index}_1').hide();
+        $.ajax({
+          url: '#{url_for(controller: controller, action: 'rpmlint_result')}',
+          data: {
+            \#{raw("'project': '#{project.to_param}',") if defined?(project)}
+            \#{raw("'package': '#{package.to_param}',") if defined?(package)}
+            \#{raw("'index': '#{index}',") if defined?(index)}
+          },
+          success: function(data) {
+            $('#result_display_#{index}_1').html(data);
+          },
+          error: function(data) {
+            $('#result_display_#{index}_1').html('<p>No rpmlint results available</p>');
+          },
+          complete: function(data) {
+            $('#result_spinner_#{index}_1').hide();
+            if ($('#result_select_link_#{index}_1').parent().hasClass('selected')) {
+              $('#result_reload_#{index}_1').show();
+            }
+          }
+        });
+      }
+
+      $('#result_display_#{index}_1').hide();
+      update_rpmlint_result();
+  :plain
+
+    function update_build_result() {
+      $('#result_spinner_#{index}_0').show();
+      $('#result_reload_#{index}_0').hide();
+      $.ajax({
+        url: '#{url_for(controller: controller, action: 'buildresult')}',
+        data: {
+          \#{raw("'project': '#{project.to_param}',") if defined?(project)}
```

Comment:
spaces around brackets:

`\#{raw("'project': '#{ project }',") if defined?(project)}`

and also in the next 3 lines.